packages/connectivity/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+##0.3.3
```

Comment:
Indent here, the markdown requires one space after the '##'.

Reply:
You also need to update `pubspec.yaml`, changing the version there to 0.3.3.

![image](https://user-images.githubusercontent.com/13699202/48660961-9b169880-ea51-11e8-9562-abd8fe2ce2b3.png)

Reply:
One extra step, as this is your first contribution, you also have to append the `plugins/AUTHORS` file with your name and e-mail address, ok?

Reply:
Done!